Cython/Compiler/Tests/TestBuiltin.py

```diff
@@ -30,3 +33,13 @@ def test_find_return_type_of_builtin_method(self):
                         self.assertTrue(hasattr(py_type, method_name), f"{type_name}.{method_name}")
                 else:
                     self.assertEqual(return_type.empty_declaration_code(pyrex=True), return_type_name)
+
+class TestBuiltinCompatibility(unittest.TestCase):
+    def test_python_builtin_compatibility(self):
+        runtime_builtin_version = frozenset(dir(__builtins__))
```

Comment:
This isn't a version.
```suggestion
        runtime_builtins = frozenset(dir(__builtins__))
```